prebuilt-ui/basic-embed/components/PrebuiltCall.js

```diff
@@ -0,0 +1,254 @@
+import DailyIframe from '@daily-co/daily-js';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
+import { writeText } from 'clipboard-polyfill';
+import { Button } from '@dailyjs/shared/components/Button';
+import {
+  Card,
+  CardBody,
+  CardFooter,
+  CardHeader,
+} from '@dailyjs/shared/components/Card';
+import Field from '@dailyjs/shared/components/Field';
+import { TextInput } from '@dailyjs/shared/components/Input';
+import { Well } from '@dailyjs/shared/components/Well';
+import { Header } from '../components/Header';
+
+export default function PrebuiltCall(props) {
+  const [demoState, setDemoState] = useState('home');
+  const [isError, setIsError] = useState(false);
+  const [roomURL, setRoomURL] = useState('');
+  const [exp, setExp] = useState();
+  const [secs, setSecs] = useState();
+  const [roomValidity, setRoomValidity] = useState(false);
+  const roomURLRef = useRef(null);
+  const iframeRef = useRef(null);
+  const callFrame = useRef(null);
+  const [linkCopied, setLinkCopied] = useState(false);
+
+  // Updates the time left that displays in the UI
+  useEffect(() => {
+    if (!exp) {
+      return false;
+    }
+    const i = setInterval(() => {
+      const timeLeft = Math.floor((new Date(exp * 1000) - Date.now()) / 1000);
+      setSecs(`${Math.floor(timeLeft / 60)}:${`0${timeLeft % 60}`.slice(-2)}`);
+    }, 1000);
+    return () => clearInterval(i);
+  }, [exp]);
+
+  // Listens for a "call" demo state, and creates then joins a callFrame as soon as that happens
+  useEffect(() => {
+    if (!iframeRef?.current || demoState !== 'call') {
+      return;
+    }
+    try {
+      callFrame.current = DailyIframe.createFrame(iframeRef.current, {
```

Comment:
nitpick: I find moving the callframe options to a constant makes it a little more readable but up to you. 
e.g. declared outside of the component:
``` 
const CALL_OPTIONS = {
showLeaveButton: true,
        iframeStyle: {
          height: '100%',
          width: '100%',
          aspectRatio: 16 / 9,
          minwidth: '400px',
          maxWidth: '920px',
          border: '0',
          borderRadius: '12px',
}
```